front/src/modules/errors/classes/ObjectMetadataNotFoundError.ts

```diff
@@ -0,0 +1,13 @@
+import { ObjectMetadataItem } from '@/object-metadata/types/ObjectMetadataItem';
```

Comment:
I feel we should remove the /classes/ folder. errors it's find by itself and aligned with our folder pattern

Reply:
I would actually move this to an /errors/ folders within object-metadata